bybit.py

```diff
@@ -10,17 +10,21 @@
 import argparse
 import threading
 from loguru import logger
-from pybit import HTTP
 
 API_LINK = 'https://api-testnet.bybit.com/v2/public/'
+PAIR = 'BTCUSD'
+LEVEL = 'L2'
 
-def get_order_book(pair):
-    """Returns L2 order book"""
-    # TODO: might want to add depth level as parameter
-    api_command = API_LINK + f'orderBook/L2?symbol={pair}'
+def get_order_book(pair=PAIR, level=LEVEL):
+    """
+    Returns L2 order book
+    TODO: might want to add depth level as parameter
```

Comment:
what does level mean? Will need depth either at this point or right before saving to storage to avoid overloading